### ISSUE TITLE
can't "add" to julia 1.0.1

### ISSUE THREAD

**Issue by JeffFessler** [ahwillia/NonNegLeastSquares.jl#12]
in 1.0.1
 ]add NonNegLeastSquares

produces  errors:
 
```
Resolving package versions...
ERROR: Unsatisfiable requirements detected for package NonNegLeastSquares [b7351bd1]:
 NonNegLeastSquares [b7351bd1] log:
 ├─possible versions are: [0.0.1, 0.1.0] or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions [0.0.1, 0.1.0]
 └─restricted by julia compatibility requirements to versions: uninstalled — no versions left
```

it installs fine in 0.7 though.

**Comment by lstagner** [comment]
The changes to make the package work for v0.7/v1.0 have not been tagged and released. You can add manually by doing `pkg> add https://github.com/ahwillia/NonNegLeastSquares.jl`

**Comment by Cody-G** [comment]
> The changes to make the package work for v0.7/v1.0 have not been tagged and released.

When might the changes be tagged and released?  We're thinking of making this package a dependency of a larger project.  Thanks!

**Comment by JeffFessler** [comment]
I found an adequate solution in the `Optim` package.

**Comment by lstagner** [comment]
I have been keeping the package up to date with Julia and everything works fine. It just needs to be added to the registry

**Comment by JeffFessler** [comment]
apologies

**Comment by MatFi** [comment]
Any estimate on when the new version will make it to the registry?

**Comment by JeffFessler** [comment]
Seems like not much activity in this repo :(
There are other NNLS methods available, e.g.:
https://github.com/StevenWhitaker/NNLS.jl

**Comment by MatFi** [comment]
but also not in the registry.. :(

**Comment by ahwillia** [comment]
I'm happy to give anyone access to this repo to maintain it in the
registry. I tried to move it over to the Julia optimizers community but
that fell through for some reason

**Comment by JeffFessler** [comment]
I am wiling to help with that.  I would also like to add code coverage and a badge for that.

**Comment by MatFi** [comment]
That would be great. The `[compat]` section in the `project.toml` needs to be updated, but that should be straightforward and I could do that, but I've never registred a Package before, so I'm not sure if I'll be a help in that process.

**Comment by ahwillia** [comment]
You both should have push access now

**Comment by JeffFessler** [comment]
thanks for adding push access.  i was able to push, but it seems like my attempts to trigger registration are not starting the registration process.  i'm unsure if this is a permission issue or if something is wrong with the bot.  i see registrations in the last hour https://github.com/JuliaRegistries/General so i suspect it is not the bot.
@ahwillia can you look at #24 and repeat that same line as another request and see if you can kick start it?
@MatFi the [compat] looked ok to me (because it seems to depend only on built-in packages i think) but if you see a needed change then submit a PR.

**Comment by JeffFessler** [comment]
well, my PR didn't seem to trigger code coverage so i've copied a github action from here:
https://github.com/simonster/Reexport.jl/blob/master/.github/workflows/CI.yml
and i hope that might do it.  anyway it might be better to migrate to github actions from travis soon.

**Comment by JeffFessler** [comment]
solved by #23 and related work.
